src/Context.h

```diff
@@ -0,0 +1,27 @@
+#ifndef TURTLEBROWSER_CONTEXT_H
+#define TURTLEBROWSER_CONTEXT_H
+
+#include "licenses/LicenseFilter.h"
+#include "licenses/LicenseModel.h"
+
+class Context {
```

Comment:
This might be a class you want to put a descriptive comment at the top explaining the essence of the class.
Otherwise it looks like the class that will grow and grow over the years (particularly if you have people besides youself submitting code).
By describing the class you set boundaries on it, limiting what people will put in it.

Reply:
It is sort of this monster 🤣 I don't know what to call it
https://github.com/turtlebrowser/turtlebrowser/pull/33#discussion_r445491794

Reply:
It is now called ApplicationState